backend/tests/modules/project_rewards/test_estimated_rewards.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from app.modules.project_rewards.service.estimated import EstimatedProjectRewards
+from tests.helpers.context import get_context
+
+
+@pytest.fixture(autouse=True)
+def before(app):
+    pass
+
+
+def test_estimated_project_rewards(mock_octant_rewards):
```

Comment:
pls, add one more test in which:
- context is taken for epoch 3
- we have at least one allocation in a db
- we have mocked matched rewards

Apart from that, lgtm

Reply:
Sure, pls take a look